Test/Sampler/sampler_input_tests.hpp

```diff
@@ -64,5 +64,39 @@ std::vector<netket::json> GetSamplerInputs() {
           {"Sampler", {{"Name", "MetropolisLocalPt"}, {"Nreplicas", 4}}}};
   input_tests.push_back(pars);
 
+  // Ising 1d with Custom Sampler
+  std::vector<std::vector<double>> sx = {{0, 1}, {1, 0}};
+  pars = {{"Graph",
+           {{"Name", "Hypercube"}, {"L", 6}, {"Dimension", 1}, {"Pbc", true}}},
+          {"Machine", {{"Name", "RbmSpin"}, {"Alpha", 1.0}}},
+          {"Hamiltonian", {{"Name", "Ising"}, {"h", 1.0}}}};
+  pars["Sampler"]["MoveOperators"] = {sx, sx, sx, sx, sx, sx};
```

Comment:
It seems that this test is causing issues in the travis-ci build (some inaccessible element in the json object) 
```
14: Assertion failed: (m_value.object->find(key) != m_value.object->end()), function operator[], file /Users/travis/build/netket/netket/External/Json/json.hpp, line 12906.
14: 
14: ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
14: test-sampler is a Catch v2.2.2 host application.
14: Run with -? for options
14: 
14: -------------------------------------------------------------------------------
14: sampler generates states in Hilbert space
14:   Sampler test (7) on {"ActingOn":[[0],[1],[2],[3],[4],[5]],"MoveOperators":[[
14:   [0.0,1.0],[1.0,0.0]],[[0.0,1.0],[1.0,0.0]],[[0.0,1.0],[1.0,0.0]],[[0.0,1.0],
14:   [1.0,0.0]],[[0.0,1.0],[1.0,0.0]],[[0.0,1.0],[1.0,0.0]]]}
```

Reply:
Yes, sorry, my mistake. The Custom Sampler allows not to specify a Sampler Name field (if MoveOperators and ActingOn fields are specified instead).

This is fixed, tests run and travis-ci builds are OK now.